### DISCUSSION TITLE
Mez Declassified: How to add Real QoL

### DISCUSSION THREAD

**Discussion by Never6** [chromiecraft/chromiecraft#7185] [Suggestions]
Disclaimer: This is probably going to be a giant post,
Recently a post was written on the chromiecraft website https://www.chromiecraft.com/en/about-the-recent-releases-on-chromiecraft/
and there is one thing in particular that I want to talk about and open a discussion here, because there are things that I or others been vocal about previously that have gotten completely ignored, or have gotten one reply from someone on the staff that completely missed the entire point to begin with. 
![image](https://github.com/chromiecraft/chromiecraft/assets/66432845/2e344fb5-65a9-44d0-acd8-4d2a6e89d4b5)
It's good that you acknowledge the decline in playerbase, and its admirable that you still continue to push through despite the lack of players.
Im hopeful that you will make the right choices when it comes to player incentive in 1. Keeping players active. and 2. Enticing New Players to come.
More population is just good in general.
Now that the fluff is out of the way, I can start with what I think you should do, Now I may miss some stuff, or people may disagree, That is ok. This is meant to be a discussion

1.  https://github.com/chromiecraft/chromiecraft/discussions/6201 I've asked for badges to be added into Karazhan twice, I think its just an overall good change, Yes, it is not blizzlike, but their are many other changes that also are not blizzlike, Someone made that point that "people would just farm badges in Karazhan and not heroics" while that is true, 1. If we are talking about blizzlike, badge of justice is supposed to cap at 200, they currently don't, so a lot of players already have tons. 2. A lot of players really can't do heroics due to lack of population, but Karazhan is really easy to find groups, especially if you are in a guild  because 1. Best in slot enchants drop from their, so any enchanter wants to run it if they don't already have those enchants. 2. It is the best farm for void crystals. 3. It is just the best way and fastest way to gear alts. The gear from Karazhan is on par (in some cases better) than heroics) Karazhan fills a similar roll to Naxx10 at 80, Its not hard, in fact can be done in greens, its mainly used as a tool to gear alts very quickly
but there are some items that can only be gotten from badges, it makes no sense to gatekeep these items from players
Not asking to add everything that comes with ZA patch, just let Karazhan drop badges

2. Account wide attunements, or honestly just remove them from the game, Attunements are an outdated mechanic and sure it's cool the FIRST time you do them, but anytime after that its just tedious and its one of the many reasons TBC is probably one of the least alt-friendly expansions. If you still standby the belief that "people should do the attunement quest" that is fine, then they should be account wide, because truthfully, and this is coming from a player who has played 2.4.3. Nobody wants to do attunements more than once, And even more so, Nobody wants to boost people through attunements, If you need to recruit, you need to attune, and players really don't enjoy doing that type of content. But you have to attune them, so players quit

3. Add an NPC that lets you turn in Badge of Justice for a Commendation Badge, This allows players to turn in their badges for items that increase their reputation with a certain faction, No brainer in my opinion

3a/4. Increase XP/Honor/Reputation/Profs 
The 2x quest XP till level 66 or whatever is nice, but imo it should be higher and it should be up till level 70

5. https://github.com/chromiecraft/chromiecraft/discussions/7183 some pvp prices just flat out make zero sense what so ever
Why does an idol cost as much as a 2Hander, Gloves are supposed to be the cheapest piece, yet they cost the same as every other piece. These prices should be looked at and reconsidered,

Thats all I have but I am hopeful that I and many others will have more ideas

**Comment by NinesCC** [comment]
> Im hopeful that you will make the right choices when it comes to player incentive in 1. Keeping players active. and 2. Enticing New Players to come.

Yes, hopefully there can be changes that bring active players back to the server and as Mez said, they don't necessarily have to be blizzlike if it adds to player engagement.

**Reply by gabonvaricel** [reply]
Adding BoJ alone to Kara will not work because the gear on the vendor right now is bad. We first need to add the ZA and Sunwell gear first.

**Reply by NinesCC** [reply]
> Adding BoJ alone to Kara will not work because the gear on the vendor right now is bad. We first need to add the ZA and Sunwell gear first.

Good point. Idk about the gear though as that could trivialize BT, but maybe something like epic gems from Sunwell Isle could be added somehow? just an idea. You can buy a epic uncut gem for 10 badges in that patch.

**Reply by Never6** [reply]
> Adding BoJ alone to Kara will not work because the gear on the vendor right now is bad. We first need to add the ZA and Sunwell gear first.

some pieces from badges are still good, i still use neck and trinkets.
ZA gear maybe, but adding sunwell gear would be a bad move IMO

**Reply by gabonvaricel** [reply]
i think a badge reset when the new vendors is also a must. cuz everyone will just prefarm all items. makes no sense

**Reply by Never6** [reply]
the weapons from sunwell badge vendor are Best in Slot and better than anything you can get in Black Temple

**Comment by Nyeriah** [comment]
Hello and thanks for your suggestion,

We are already considering our options in order to make the game more enjoyable as well as more friendly to alts and new players, as mentioned in the official statement.

More information to come in the upcoming weeks.

Regards.
